### ISSUE TITLE
value polling interval

### ISSUE THREAD

**Issue by algar42** [ioBroker/ioBroker.ham#6]
In addition to value polling interval is it possible to add a polling interval delay setting ( in milliseconds), so the device states are checked with "polling interval" but with "delay" between requests? It will be useful for some 433 MHz devices when a lot of channels being requested at once makes difficult to use the same frequency by other devices.

**Comment by Apollon77** [comment]
Hm .... would not be the homebridge plugin needs to make sure it handles that? When you open a „room“ in the iOS home app also homebridge requests all Infos for all devices in this room. So a „mass“ request also happens there.

**Comment by Apollon77** [comment]
Which homebridge-plugins do you use to integrate those 433MHz devices?

**Comment by algar42** [comment]
it is homebridge-noolite .

**Comment by Apollon77** [comment]
found in forum :-) Please see there. open an isue with tem and we see what they say :)

**Comment by algar42** [comment]
thanks ;)

**Comment by algar42** [comment]
just a question. When we use global-homebridge-mode is there a way to initiate device status update from iobroker?

**Comment by Apollon77** [comment]
Did you open an issue with noolite plugin?

For the global mode question: I also thought about it but then checked things and decided to wait till someone asks because a bit more difficult there ;-)
You can open me an issue if you like to have this and unwilling check as I find time next.

**Comment by algar42** [comment]
No, I didn't raise any issues to homebridge noolite yet because I faced another problem. When not in global-mode, after some time MTRF-64 stops responding to status updates (even from plugin web interface). I see that request for update is sent to MTRF, but nothing received back to plugin. I currently can't understand where the problem is (USB-TTL, linux settings, plugin itself or wrapper mode) and moved to global-mode to see. Will check tomorrow if it still receives messages back from MTRF or not.... Currently it is more important than "mass attack" to 433 MHz :)
(it is why I actually asked about possibility to update states in global-mode).
